src/Strategy/Python/Pipenv.hs

```diff
@@ -302,3 +319,25 @@ instance FromJSON PipenvGraphDep where
       <*> obj .: "installed_version"
       <*> obj .: "required_version"
       <*> obj .: "dependencies"
+
+---------- Pipfile
+
+data PipfileToml = PipfileToml
+  -- {pipfilePackages :: Map Text (), pipfileDevPackages :: Map Text ()}
+  {pipfilePackages :: Map Text PipfilePackageVersion, pipfileDevPackages :: Map Text PipfilePackageVersion}
```

Comment:
If you haven't, could you run `make fmt-ci`? 

I forget if it's a rule, but we typically put each field definition on its own line.

Reply:
I do have the formatter set up and it seems to be happy with everything on one line in this case. But I'll change it anyway.